onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -377,6 +377,27 @@ ONNX_MS_OPERATOR_SET_SCHEMA(FastGelu, 1,
                                   return true;
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(RelPosAttnBiasGen, 1,
+                            OpSchema()
+                                .SetDoc("Compute binned relative position bias")
+                                .Attr("max_distance", "Max distance", AttributeProto::INT)
+                                .Attr("is_bidirectional", "Default value is 0.", AttributeProto::INT, static_cast<int64_t>(0))
+                                .Input(0, "bias_table", "2D input tensor with shape (num_buckets, num_heads), COL-major(See UT for example)", "T")
```

Comment:
Can bias table be generated? I saw some kernel to do that in FT. If so, probably we can mark it as optional, and use kernel to generate the table if not provided.

Reply:
For T5, I saw https://github.com/huggingface/transformers/blob/35a7052b61579cfe8df1a059d4cd3359310ec2d1/src/transformers/models/t5/modeling_t5.py#L380-L442

Reply:
do you mean this: https://github.com/huggingface/transformers/blob/35a7052b61579cfe8df1a059d4cd3359310ec2d1/src/transformers/models/t5/modeling_t5.py#L433

it's handled here: https://github.com/microsoft/onnxruntime/pull/14149/files#diff-691e372f711bad0f23eb0f350802f7a4656ce33aec3e428d99be34dc37032418R43